### PR TITLE
Move tests to azure pipelines

### PR DESCRIPTION
**Pull request by odinuv**
- testy presunuty na azure pipelines
- Rozlozeni testu:
  - z common jsem oddelil DownloadTablesDefaultTest a DownloadFilesTest, ktery trvaji dlouho a maji docela ohranicenej scope (stacilo upravit nazev jednoho bucketu a tagu), ostatni jsem nechal (uz se mi s tim nechce drbat)
  - common se sklada z common1, files a common2 a tyhle 3 muzou bezet parallene (v jednom projektu, protoze to testy umoznuji)
  - common se spousti zvlast na aws a azure a muze bezet parallelne (v ruznych projektech)
  - ke common1 jsou pro aws priplacnuty aws specificke testy a pro azure priplacnuty azure specificke testy
  - na azure jsou jeste synapse testy, ktere ale taky muzou bezet parallene, protoze bezi jeste v jinem projektu
- ^ toto vsechno se provadi seriove pro PHP5.6 a PHP7.4 (pouziva stejny token)

K tomu je jeste: `trigger: batch: true` ktery zajistuje, ze cela pipeline se pousti seriove. A do testu jsem pridal log storage klienta, aby bylo videt, co se kde deje (testy ted bezi na 3 ruznych projektech).

https://dev.azure.com/keboola-dev/input-mapping/_build/results?buildId=1883&view=logs&j=c5573706-b422-504a-bf2e-41fa9a92859d

https://keboola.atlassian.net/browse/PS-876

https://travis-ci.org/github/keboola/input-mapping/builds/720592648

edit: znamena to teda, ze je to slo z ~100 minut na ~35 minut

### CODE REVIEW

**Comment by pivnicek** [comment]
Not sure if you want to do it here, but in the sandboxes-jupyterlab I'm creating dockerfiles with concatenation.  So for example is you can have 1 Dockerfile with everything except the from line and do something like `echo "FROM php:7.4-cli" | cat Dockerfile > Dockerfile74`.  but not really worth it I guess since there's just the 2.

**Comment by odinuv** [comment]
i'd prefer not to generate dockerfiles dynamically during build :)

**Comment by pivnicek** [comment]
fair enough :-)

File: docker/php/php.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+; Maximum amount of memory a script may consume (128MB)
+; http://php.net/memory-limit
+memory_limit = -1
+
+; Defines the default timezone used by the date functions
+; http://php.net/date.timezone
+date.timezone = UTC
```
Comment (pivnicek) [inline docker/php/php.ini:7]: Is this actually going to change anything in a functional sense?

**Review by pivnicek — APPROVED** [review]
Huge improvement! 🍻

File: docker/php/php.ini (hunk 1 above)
Comment (odinuv) [reply docker/php/php.ini:7]: no, i just copied the "standard" setup from php component

**Review by MiroCillik — COMMENTED** [review]
Dobra praca 👍  🎖️

**Pull request merged**